src/imageGpsExifRemover.ts

```diff
@@ -17,27 +17,29 @@ const END_OF_PNG_TAG = 0x49454e44
 const PNG_ITXT_TAG = 0x69545874
 const PNG_XMP_TAG = 'XML:com.adobe.xmp'
 
+import { Logger } from './logger'
+import { getString } from './gpsRemoverHelpers'
+
 
-const killGPS
-= async (startOffset: number, littleEndian: boolean, read: ReadFunction, write: WriteFunction) => {
+const killGPS = async (startOffset: number, littleEndian: boolean, read: ReadFunction, write: WriteFunction) => {
   const entryNumberDataView = await readNextChunkIntoDataView(2, startOffset, read)
   const entries = entryNumberDataView.getUint16(0, littleEndian)
   const entryOffset = startOffset + 2
   // kill 'em all
-  console.log('writing 0s on gps data...', entries, entryOffset)
+  Logger.debug('writing 0s on gps data...', entries, entryOffset)
```

Comment:
might be a good idea to just take the logs out at this point, I was mostly just using them to figure out how this stuff worked when I was writing it